source/facter/3.1/release_notes.md

```diff
@@ -5,19 +5,65 @@ title: "Facter 3.1: Release Notes"
 
 This page documents the history of the Facter 3.1 series. (Previous version: [Facter 3.0 release notes](../3.0/release_notes.html).)
 
-Facter 3.1.0
------
+## Facter 3.1.1
+
+Released October 29, 2015.
+
+Shipped in puppet-agent version 1.2.7.
+
+Facter 3.1.1 fixes several bugs and introduces support for AIX 5.3, 6.1, and 7.1, and Solaris 11.
+
+* [Fixes for Facter 3.1.1]()
+* [Introduced in Facter 3.1.1]()
+
+### New Supported Platforms: AIX and Solaris 11
+
+Facter 3.1.1 supports AIX 5.3, 6.1, and 7.1 on PowerPC, and Solaris 11 on x86 and SPARC.
+
+* [FACT-890](https://tickets.puppetlabs.com/browse/FACT-890): Facter 3 AIX Support
```

Comment:
Since there were actual facter code changes to support AIX and Solaris 11, e.g. FACT-549, FACT-799, FACT-1232, I'm :+1: for mentioning we've added support here (as opposed to only at the higher level puppet-agent).